rules/obfuscation/js/char_codes.yara

```diff
@@ -13,9 +13,12 @@ rule child_process : critical {
 		$a_shift = "shift"
 		$a_push = "push"
 
-		$const = "const "
-		$function = "function("
-		$return = "{return"
+		$a_const = "const "
+		$a_function = "function("
+		$a_return = "{return"
+
```

Comment:
also mark this as a "high"

Reply:
If we move these to high, I can remove the exclusions. That would be a lot cleaner.

Reply:
Works for me. In my mind, "CRITICAL" rules should never be broad enough for false positives to happen.